theme6/source/index.html.erb

```diff
@@ -1,23 +1,112 @@
 ---
-title: Welcome to Middleman
+title: Lil Peep
 ---
 
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 400 340" class="middleman-logo" aria-labelledby="middleman-logo__title" role="img">
-  <title id="middleman-logo__title">Middleman</title>
-  <path class="middleman-logo__top-left-bar" fill-opacity=".45" d="M0 40L200 0v30L0 60z"/>
-  <path class="middleman-logo__top-right-bar" fill="#fff" d="M200 0l200 40v20L200 30z"/>
-  <path class="middleman-logo__left-m" fill-opacity=".45" d="M0 78v184l45 5V152l45 83 47-83v129l53 7V52l-57 8-43 83-43-70z"/>
-  <path class="middleman-logo__right-m" fill="#fff" d="M400 78v184l-45 5V152l-45 83-47-83v129l-53 7V52l57 8 43 83 43-70z"/>
-  <path class="middleman-logo__bottom-left-bar" fill-opacity=".45" d="M0 300l200 40v-30L0 280z"/>
-  <path class="middleman-logo__bottom-right-bar" fill="#fff" d="M200 340l200-40v-20l-200 30z"/>
-</svg>
-
-<h1>
-  Middleman is Running
-</h1>
-
-<%= link_to(
-  "Read Documentation",
-  "https://middlemanapp.com/basics/templating_language/",
-  target: "_blank"
-) %>
+<section class="featured-products">
+  <h2>FEATURED TOYS</h2>
+  <div class="featured-products-list">
+    <div class="product-card first-product-card">
+      <div class="products-slider">
+        <div class="front-label">
+          <p>NEW</p>
+        </div>
+        <div class="slider-navigator slider-navigator-left">
+          <a href="#" onclick="changeImage(-1)">
+            <img src="images/arrow_left.png" alt="Arrow left">
+          </a>
+        </div>
+        <div class="product-image">
+          <img id="featuredToy" src="images/bear_toy.png" alt="Clown Bear Toy">
+        </div>
+        <div class="slider-navigator slider-navigator-right">
+          <a href="#" onclick="changeImage(1)">
+            <img src="images/arrow_right.png" alt="Arrow right">
+          </a>
+        </div>
+      </div>
+      <div class="product-data">
+        <p>Clown Bear - Plush Circus Series</p>
+        <div class="price">
+          <p class="price">$5 USD</p>
+          <p class="offer">$10 USD</p>
+        </div>
+        <button type="button" name="view-button">VIEW</button>
+      </div>
+    </div>
+    <div class="product-card">
+      <div class="products-slider">
+        <div class="front-label">
+          <p>NEW</p>
+        </div>
+        <div class="slider-navigator slider-navigator-left">
+          <a href="#" onclick="changeImage(-1)">
```

Comment:
Let's try to avoid using direct JS functions inside HTML code, instead of this, create a CSS class, and using JS set a listener to that class or ID, I prefer ID when there are going to be listeners attached to the element and then call the `changeImage`  function. Something like this:

```
<a href="#" id="left-arrow (or whatever name you prefer)">...</a>
```
In JS
```
const leftArrow = document.getElementById('the id you chose');
leftArrow.addEventListener('click', () => {
  changeImage(-1);
});
```
Something like that...

Reply:
Another thing... semantically speaking that should be a `<button>` instead of an anchor `<a>`, anchors should be only used to jump between sections or pages, buttons are used more to trigger different actions on the current page